pkg/tool/oc.go

```diff
@@ -19,9 +19,9 @@ func NewOc(exec exec.ProcessExecutor) Oc {
 
 const osVersionKey = "openshiftVersion"
 
-func (o Oc) GetVersion() (string, error) {
+func (o Oc) GetVersion(ov string) (string, error) {
```

Comment:
This change is not required; to use the value coming from the command line as fallback just do that in the caller.

Reply:
Reverted. 03bd100976c1685bdec2a689611f61d3e2371157